modules/byoh-removal.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/creating_windows_machinesets/byoh-windows-instance.adoc
+
+[id="removing-byoh-windows-instance"]
+= Removing BYOH Windows instances
+You can remove BYOH instances attached to the cluster by deleting the instance's entry in the config map. Deleting an instance reverts that instance back to their state prior to adding to the cluster. Any logs and container runtime artifacts are not added to these instances.
+
+For an instance to be cleanly removed, it must be accessible with the current private key provided to WMCO. For example, to remove the `10.1.42.1` instance from the previous example, the config map would be changed to
+the following:
```

Comment:
Tiny nit: this could be wrapped with the rest of the containing sentence on line 9.